examples/client/client.vcxproj

```diff
@@ -42,46 +42,46 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v110</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
We cannot accept the visual studio project changes. Those must be reverted.

Reply:
I understand the need to continue to have the VS2012 files for existing customers. Any objection to including _additional_ project files that target more modern, such as VS2022? Perhaps a naming convention of simply adding `_VS2022` to the project file name?

Reply:
It is great you're continuing to support older versions of the VS toolchain such as VS2012 and VS2005. Though it was a very big red flag to me to see reliance on a decade old complier & standard library as the default selection as the primary Windows build path in a security library. 

As the VS2012 toolchain is no longer available outside specific MSDN subscriptions, I suggest you create additional 2012 project files for legacy users and update to a more modern toolchain that will make it easier for the majority of new Windows users to work with your library. This has the additional benefit of clearly surfacing the v110 build chain support. 

I have updated the PR as suggested, targeting the v143 toolchain. This can be used from the most recent versions of Visual Studio, which automatically prompts the user to download the toolchain if it isn't installed. 

Note, sslSniffTest was not changed as it targets the v141 toolchain (VS2017), and is still [readily available](https://visualstudio.microsoft.com/vs/older-downloads/).